src/main/scala/firrtl/graph/DiGraph.scala

```diff
@@ -48,7 +53,7 @@ class DiGraph[T] private[graph] (private[graph] val edges: LinkedHashMap[T, Link
     * @param v the specified node
     * @return a Set[T] of all vertices that v has edges to
     */
-  def getEdges(v: T): Set[T] = edges.getOrElse(v, Set.empty)
+  def getEdges(v: T, prevOpt: Option[collection.Map[T, T]] = None): Set[T] = edges.getOrElse(v, Set.empty)
```

Comment:
One of my main comments with the original PR was about the addition of this extra parameter to `getEdges`.

This parameter is only used in `BFS`, and it fundamentally changes the behavior of the underlying implementation of BFS. My thought is that it would make more sense to just rely on overloading `BFS` in the derived class or something like that.